dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sfdr/categories/environmental/subcategories/SfdrEnvironmentalEmissions.kt

```diff
@@ -0,0 +1,20 @@
+package org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories
+
+import org.dataland.datalandbackend.model.DataPoint
+import org.dataland.datalandbackend.model.DataPointWithUnit
+import org.dataland.datalandbackend.model.enums.commons.YesNo
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * Fields of the subcategory "Emissions" belonging to the category "Environmental" of the sfdr framework.
+ */
+data class SfdrEnvironmentalEmissions(
+    val inorganicPollutants: DataPointWithUnit<BigDecimal>? = null,
+
+    val airPollutants: DataPointWithUnit<BigDecimal>? = null,
+
+    val ozoneDepletionSubstances: DataPointWithUnit<BigDecimal>? = null,
```

Comment:
Why do you have to manually set the classes of these three fields to `DataPointWithUnit<BigDecimal>` ?

If you insert these types directly into the Excel and make the devtools handle those, then they would automatically be inserted here.

What do you think?

Reply:
Dev tools now gives the correct type (DataPointWithUnit) back

Reply:
👍